modules/apps/portal-security/portal-security-content-security-policy/src/main/java/com/liferay/portal/security/content/security/policy/internal/servlet/filter/ContentSecurityPolicyFilter.java

```diff
@@ -181,6 +181,14 @@ private boolean _isExcludedURIPath(HttpServletRequest httpServletRequest) {
 			return false;
 		}
 
+		for (String s : _INTERNALLY_EXCLUDED_URI_PATHS) {
```

Comment:
Variable should be try to match the collection element. Maybe call it `internallyExcludedUriPath`?

Reply:
So do you think I should use it in single form? I decided to use the plural because the one from the config interface is `excludedURIPaths` so I thought it would make sense to use them both in plural.

Reply:
I think singular makes more sense in the for loop for the `s` variable which is only one element of the array. In the configuration it is a String array so that should be plural.

Reply:
I misunderstood you comment. I thought you were talking about `_INTERNALLY_EXCLUDED_URI_PATHS ` but now I see you were talking about `s`. It makes sense. I rename it. Thanks!